server.py

```
@@ -59,35 +61,96 @@ def main():
+                    # Restart JS8Call connection if it died
+                    if not js8call_client.connected and js8call_client.db_conn:
+                        js8_thread = threading.Thread(target=js8call_client.connect, daemon=True)
+                        js8_thread.start()
+
+                logging.info("Connected to Meshtastic interface.")
+
+                # Main wait loop - monitoring connection if possible
+                while True:
+                    # 1. Update heartbeat file ONLY if we are supposedly connected
+                    try:
+                        with open('/tmp/bbs_heartbeat', 'w') as f:
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/bbs_heartbeat"

(S108)